.github/workflows/build.yml

```diff
@@ -119,6 +119,55 @@ jobs:
       - name: Test
         run: cd build && ctest -C Release -j3 --output-on-failure
 
+  freebsd:
+    name: FreeBSD
+    runs-on: ubuntu-latest
+    if: github.event_name != "pull_request"
```

Comment:
presume trying to save some CPU time?

FWIW we could also use `cirrus CI` which provides native FreeBSD runners (hosted in google cloud) instead

Reply:
True. But I don't see a compelling need to switch, for now.

We'd still need a VM for Solaris.

Reply:
cirrusci provides their own VMs which perform better than GitHub's